builder/builder-next/controller.go

```diff
@@ -251,3 +253,15 @@ func parsePlatforms(platformsStr []string) ([]specs.Platform, error) {
 	}
 	return out, nil
 }
+
+func getEntitlements(conf config.BuilderConfig) ([]string, error) {
+	var ents []string
+	// Incase of no config settings, NetworkHost should be enabled & SecurityInsecure must be disabled.
+	if conf.Entitlements.NetworkHost == nil || (conf.Entitlements.NetworkHost != nil && *conf.Entitlements.NetworkHost == true) {
```

Comment:
Removing `conf.Entitlements.NetworkHost != nil && ..` will make only way to enable Network-host is by no entry in config file.

```json
{
        "features": {
                "buildkit": true
        },
        "builder":{
                "entitlements":{
                        "network-host":true,
                        "security-insecure":false
                        }
        }
}
```
The above config file will also not enable `network-host`. I think that will be wrong behaviour.

Reply:
Sorry, I only meant the `conf.Entitlements.NetworkHost != nil &&` part not `*conf.Entitlements.NetworkHost == true` after it that is still needed. Just as small cleanup because you already check if it is nil before with no functional change.